extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -486,6 +494,37 @@ export class AmpIframe extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Listen for Pym.js messages for 'height' and 'width'.
+   *
+   * @see http://blog.apps.npr.org/pym.js/
+   * @param {!MessageEvent} event
+   * @private
+   */
+  listenForPymMessage_(event) {
+    if (!this.iframe_ || event.source !== this.iframe_.contentWindow) {
+      return;
+    }
+    const data = getData(event);
+    if (typeof data !== 'string' || 'pym' !== data.substr(0, 3)) {
```

Comment:
We have a `startsWith` helper

Reply:
Utilized in af98c2240964d85ed2636a916b17f3bc2c046b60